packages/ui/src/primitives/tailwind/Tabs/index.tsx

```diff
@@ -110,7 +110,17 @@ const Tabs = ({
         ))}
         {tabsData[currentTab.value]?.rightComponent}
       </div>
-      {scrollable ? tabsData.map((tab, index) => tab.bottomComponent) : tabsData[currentTab.value]?.bottomComponent}
+      {scrollable
+        ? tabsData.map((tab, index) => {
+            return (
+              <>
+                <ErrorBoundary key={index} fallback={<></>}>
+                  <Suspense fallback={<></>}>{tab.bottomComponent}</Suspense>
+                </ErrorBoundary>
+              </>
+            )
+          })
+        : tabsData[currentTab.value]?.bottomComponent}
```

Comment:
we should wrap the error boundary and suspense inside the `ReactorDebug` or make a higher order component such `ReactorDebugWrapper` with the error boundaries and suspense in it

Reply:
we might need to add error boundaries when there is a single element as well in that case.
we can map over the [`DebugTabs` values](https://github.com/ir-engine/ir-engine/pull/1112/files#diff-e7c3ea17cd35c689bea85b17184731bd08283c5328b6e4c709e2904452c9c932R75-R80) and wrap them with error boundary and suspense.